testsuite/features/secondary/srv_content_lifecycle.feature

```diff
@@ -156,4 +156,4 @@ Feature: Content lifecycle
     And I follow "clp_name"
     And I click on "Delete"
     And I click on "Delete" in "Delete Project" modal
-    Then I should see a "There are no entries to show." text
+    Then I should not see a "clp_name" text
```

Comment:
Wouldn't there be a banner message indicating the  successful deletion of the project with its name?

Reply:
This is how the UI looks when you delete a CLM project:
![image](https://user-images.githubusercontent.com/12104291/202456035-c1621b7b-177d-4bba-a6fc-7d8f1ee5587c.png)

Reply:
If the step does not detect that banner text it should be alright then , otherwise it might fail.

Reply:
Only the label is displayed in this banner so it should not interfere with the step, everything is fine.